### PR TITLE
Updated ctestis dox and fixed bug

### PR DESCRIPTION
**Pull request by lprichard**
Added documentation on how to derive an appropriate sky value and scaling of counts to the correct level for the code and updated parameter descriptions. Also added information on implementing code for a single cosmic-ray split image. Fixed bug when stisimage filename is not provided.

### CODE REVIEW

File: stistools/ctestis.py [hunk 1]
```diff
@@ -177,7 +204,7 @@ def ctestis(ycol, net, sky, stisimage=None, mjd=None, nread=None,
     dy512 = 0.025 * cti10000 - (0.00078 * cti10000 * cti10000)
 
     # prep ycol
-    if sx2 or "_sx2" in stisimage:
+    if sx2 or ((stisimage!=None) and ("_sx2" in stisimage)):
```
Comment (sean-lockwood) [inline stistools/ctestis.py:207]: Comparison to `None` should use `is`:

```python
if sx2 or ((stisimage is not None) and ("_sx2" in stisimage)):
```

File: stistools/ctestis.py [hunk 2]
```diff
@@ -22,6 +22,30 @@
...
+a sky-background estimate, and the source Y-position on the detector 
+(since CTI effects worsen furthest from the readout). The single-pixel 
+sky-background estimate should be measured from individual cosmic-ray (CR) 
+split, bias- and dark-subtracted, and flat-fielded images (flt.fits) that 
+have not had any sky subtracted. This can determined with random sampling 
+and/or iterative sigma-clipping of sky pixels (e.g., Goudfrooij et al. 
+2006). The net counts measured from the science images (summed, sky-
+subtracted exposures) should then be scaled to the exposure time of the CR
+split FLT image (e.g., if CRSPLIT=5, the net counts are divided by five).
+Note, that not all FLT files have equal exposure times so a fractional
```
Comment (sean-lockwood) [inline stistools/ctestis.py:35]: Wording is a bit confusing.  Perhaps:
```
Note, that not all extensions composing an FLT have equal exposure times...
```

File: stistools/ctestis.py [hunk 3]
```diff
@@ -22,6 +22,30 @@
...
+(since CTI effects worsen furthest from the readout). The single-pixel 
+sky-background estimate should be measured from individual cosmic-ray (CR) 
+split, bias- and dark-subtracted, and flat-fielded images (flt.fits) that 
+have not had any sky subtracted. This can determined with random sampling 
+and/or iterative sigma-clipping of sky pixels (e.g., Goudfrooij et al. 
+2006). The net counts measured from the science images (summed, sky-
+subtracted exposures) should then be scaled to the exposure time of the CR
+split FLT image (e.g., if CRSPLIT=5, the net counts are divided by five).
+Note, that not all FLT files have equal exposure times so a fractional 
+scaling of the CR split to total exposure time of the science image should
```
Comment (sean-lockwood) [inline stistools/ctestis.py:36]: "to total exposure time of the CR-combined science image (e.g. CRJ) should"

**Comment by sean-lockwood** [comment]
@lprichard -
It would be nice to get this PR merged into Master in time for the eminent `stistools` release needed to fix the Astropy import issue.  Let me know if you'd prefer I submit a PR to your fork with the proposed changes.

**Comment by lprichard** [comment]
Apologies, I'd missed I had to do something else. I agree with all of these changes so please go ahead and merge them in. Thanks!

**Comment by sean-lockwood** [comment]
PR with changes to merge into this fork has been submitted.

**Pull request merged**